CSLabs.Api/Models/Maintenance.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CSLabs.Api.Models
+{
+    public class Maintenance
+    {
+        public int Id { get; set; }
+        public bool IsMaintenanceMode { get; set; }
+        public bool IsRestorationTimeKnown { get; set; }
+        public int RestorationTime { get; set; }
```

Comment:
You should use the `DateTime` property to denote a date/time. 
`IsRestorationTimeKnown`  can be determined by checking if `RestorationTime` is not null and thus is not needed to store in the database.
```suggestion
        public DateTime? RestorationTime { get; set; }
```

Reply:
You're right, this is a better solution. Some small adjustments will need to be made in the frontend to reflect this, but in all this is definitely cleaner.